lib/gui/app.js

```diff
@@ -87,7 +87,9 @@ app.run(() => {
 });
 
 app.run((ErrorService) => {
-  analytics.logEvent('Application start');
+  analytics.logEvent('Application start', {
+    packageType: packageJSON.packageType
```

Comment:
Should this also log the application version? AFAICT from reading through the code, the version is only logged by the 'Checking for updates' event, so if we instead hit the 'Not checking for updates' event (i.e. `(!shouldCheckForUpdates || !updatesEnabled)`) then the `currentVersion` doesn't get logged?

Also, something I've been wondering for a while is whether the `console.log` on line 76 should also report the `currentVersion`, so that when people copy'n'paste us their DevTools logs, we can see from the logs what version they're running. And if you think that's a good idea, then it's probably worth adding this `packageType` info in there too?